testbed/core/models/outbox.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from .actor import Actor
+from .activity import CreateActivity, LikeActivity, FollowActivity
+
+class PortabilityOutbox(models.Model):
+    actor = models.ForeignKey(
+        Actor, on_delete=models.CASCADE, related_name="portability_outbox"
+    )
+    activities_create = models.ManyToManyField(CreateActivity, related_name="outboxes")
+    activities_like = models.ManyToManyField(LikeActivity, related_name="outboxes")
+    activities_follow = models.ManyToManyField(FollowActivity, related_name="outboxes")
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"Outbox for {self.actor.username}"
+
+    # Helper method to add any type of activity
+    def add_activity(self, activity):
+        if isinstance(activity, CreateActivity):
+            self.activities_create.add(activity)
+        elif isinstance(activity, LikeActivity):
+            self.activities_like.add(activity)
+        elif isinstance(activity, FollowActivity):
+            self.activities_follow.add(activity)
+
+    def get_json_ld(self):
```

Comment:
I think one of the reasons why you are finding all your models making the models.py file so long already, is because the ActivityPub objects all contain views within them.  The "get_json_ld" method on the model is a view method.  If ActivityPub or LOLA included a second way of representing a PortabilityOutbox, perhaps a "summary view" - would you write "get_json_ld_summary_view"?  I think then it would become really clear if  more than one view but for the same object.

Reply:
This makes sense! I'll work towards that.

The `get_json_ld` methods are indeed view/serialization logic mixed with the models.  I know we still have some work to do with serialization and mix-in but do you think a better approach would be to move these to serializers in the meantime?